app-office/moneydance/files/moneydance-bin

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Find JAVA
+JAVA_OPTS="-client -Dawt.useSystemAAFontSettings=gasp -Dawt.useSystemAAFontSettings=on -Xmx2048m"
+MONEYDANCE_LIBS="/usr/share/moneydance/lib"
+OPENJDK11="/usr/lib/jvm/openjdk-11"
+OPENJDK11BIN="$(find /opt -name 'openjdk-bin-11.*' -type d | head -n1)"
+
+# Choose JAVA
+if [ -d "${OPENJDK11BIN}" ]; then
+	JAVA_HOME="${OPENJDK11_BIN}"
+elif [ -d "${OPENJDK11}" ]; then
+	JAVA_HOME="${OPENJDK11}"
+else
+	echo "Error: No installed JAVA found!"
```

Comment:
You may want to be more specific (`Java 11` maybe?).

Reply:
Added.